templates/monorepo/apps/azure-functions-api/src/utils/cosmos/dependency.ts

```diff
@@ -0,0 +1,5 @@
+import { Database } from "@azure/cosmos";
```

Comment:
I'd postpone any integration to another PR / step

Reply:
Remove database dependency will make the health-check only an HTTP Api call. I agree that we can update this part on another PR.